src/gdcdictionary/schemas/annotation.yaml

```diff
@@ -310,6 +310,12 @@ links:
         target_type: submitted_aligned_reads
         multiplicity: many_to_many
         required: false
+      - name: submitted_expression_arrays
```

Comment:
Should this name be submitted_expressions? (the plural indicates a collection of values)

Reply:
I think `submitted_expression_arrays` is correct, since this would be the backref from annotation to find the edges it is connected to, i.e. for case it is `cases`:

```
In [24]: g.nodes(Annotation).first().cases
Out[24]: [<Case(b5622ca2-8f51-453e-b411-b2ac045bb04a)>]
```